en/camera/index.md

```diff
@@ -1 +1,99 @@
 # Cameras
```

Comment:
@julianoes I did a big update today, but I've now had yet another epiphany: **you should not need the camera driver** for a MAVLink camera integration. 

PX4 basically handles mavlink camera protocol commands in missions (and the older commands) by emitting the mission item as a command addressed to component id 100. It handles incoming commands by forwarding them to the camera.

So what does the driver do in this scenario? Essentially for MAVLink camera backend it just emits [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) for some, but not all camera trigger events.
That should be done by the camera. 

So what we should do is make sure that the camera protocol requires [CAMERA_TRIGGER](https://mavlink.io/en/messages/common.html#CAMERA_TRIGGER) to be emitted on image capture, and then you wouldn't need the driver at all.

Am I right?

Reply:
> Are there old MAVLink cameras that just support the old API for which using the backend makes sense? 

I think most simple triggering cameras are like that. I'm blanking on the name of an example that I have in mind.

Reply:
1. Depends on the definition of a "MAVLink camera". It just comes down to what the camera supports. I'd assume most implement the ArduPilot camera messages like DIGICAM_...
2. Don't know.
3. No idea. It's probably just incomplete.
4. I don't think it's deliberate. Was just always an incomplete/tricky topic.